blueprints/secops/bitwarden-cli/ops2deb.yml

```diff
@@ -0,0 +1,13 @@
+name: bitwarden-cli
+version: 2024.6.1
+homepage: https://bitwarden.com/help/cli/
+summary: Tool to access and manage a bitwarden vault
+description: |-
+  The Bitwarden command-line interface (CLI) is a powerful, fully-featured
```

Comment:
No need to repeat the summary, you can start at powerful and fully-featured

Reply:
Done.